src/hott/06-contractible.rzk.md

```diff
@@ -338,51 +340,51 @@ For example, we prove that based path spaces are contractible.
 
 -- Based path spaces are contractible
 #def is-contr-based-paths
-  (A : U)         -- The ambient type.
-  (a : A)         -- The basepoint.
-  : is-contr (Σ (x : A) , a = x)
-  := (based-paths-center A a , based-paths-contracting-homotopy A a)
+  ( A : U)         -- The ambient type.
+  ( a : A)         -- The basepoint.
+  : is-contr (Σ ( x : A) , a = x)
+  := ( based-paths-center A a , based-paths-contracting-homotopy A a)
 ```
 
 ## Contractible products
 
 ```rzk
 #def is-contr-product
-  (A B : U)
-  (AisContr : is-contr A)
-  (BisContr : is-contr B)
+  ( A B : U)
+  ( A-is-contr : is-contr A)
+  ( B-is-contr : is-contr B)
   : is-contr (product A B)
   :=
-    ( (first AisContr , first BisContr) ,
+    ( (first A-is-contr , first B-is-contr) ,
       \ p -> path-product A B
-              ( first AisContr) (first p)
-              ( first BisContr) (second p)
-              ( second AisContr (first p))
-              ( second BisContr (second p)))
+              ( first A-is-contr) (first p)
+              ( first B-is-contr) (second p)
+              ( second A-is-contr (first p))
+              ( second B-is-contr (second p)))
 
 #def first-is-contr-product
-  (A B : U)
-  (AxBisContr : is-contr (product A B))
+  ( A B : U)
+  ( AxB-is-contr : is-contr (product A B))
```

Comment:
`is-contr-...`

Reply:
Same question as above to @fredrik-bakke

Reply:
Done